src/component/Details.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useLocation, Link } from 'react-router-dom';
+
+const Details = () => {
+  const location = useLocation();
+  const {
+    // eslint-disable-next-line
+    name, image, description, country,
+    // eslint-disable-next-line
```

Comment:
- [ ] Kindly remove all `Eslint` disabling comments from this file. `Eslint` helps us to write clean and readable Javascript code. Using disabling comments defeats this purpose. 👍